src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import ArticlesCards from "./ArticlesCards";
+
+const ArticlesList = () => {
+  const [articlesList, setArticlesList] = useState([]);
+
+  useEffect(() => {
+    axios.get("https://nc-be-project.onrender.com/api/articles").then((res) => {
```

Comment:
extract this request to a separate function in a utils file to separate concerns